internal/debug/cmd_version.go

```diff
@@ -0,0 +1,23 @@
+package debug
+
+import (
+	"fmt"
+
+	"github.com/altipla-consulting/wave/internal/query"
+	"github.com/spf13/cobra"
+)
+
+var cmdVersion = &cobra.Command{
+	Use:     "version",
+	Short:   "Debug the version that is taken from a project.",
```

Comment:
No existe nada llamado "proyecto" en wave:
```suggestion
	Short:   "Debug the deployment version.",
```